lytics.js

```diff
@@ -2,7 +2,7 @@ import * as Attributes from './utils/attributes';
 import * as Listeners from './utils/listeners';
 
 const noopTracker = function (props, cb) {
```

Comment:
What's the use of props here?

Reply:
Props contain the lytics properties of the clicked element.